README.md

```diff
@@ -44,28 +50,39 @@ command line like this:
 ./normalizer < sample.csv > output.csv
 ```
 
+### JSON:
+
+Please write a static webpage that displays the data contained in
+`sample.json`. You should write JavaScript (or code that compiles
+to JavaScript) that takes the sample data and formats it according
+to the normalization rules listed below. Don't worry about making
+it too fancy -- you can display and style the data however you wish,
+but we will be looking for valid markup.
+
+### Normalization rules:
+
 Normalized, in this case, means:
 
-* The entire CSV is in the UTF-8 character set.
-* The `Timestamp` column should be formatted in RFC3339 format.
-* The `Timestamp` column should be assumed to be in US/Pacific time;
+- The entire CSV is in the UTF-8 character set.
```

Comment:
Replace ~CSV~ with `data file`

Reply:
^^ this but I'd rather see `data` alone or else `data set` if that makes more sense.